src/main/kotlin/apply/application/mail/MailData.kt

```diff
@@ -24,15 +25,18 @@ data class MailData(
     @field:NotNull
     var sentTime: LocalDateTime = LocalDateTime.now(),
 
+    var attachFiles: Map<String, ByteArrayResource> = emptyMap(),
+
     @field:NotNull
     var id: Long = 0L
```

Comment:
`TermSelectData`에서는 필드 파라미터 배치를 이렇게 두고 있더라구요! 
id에 NotNull Validation을 붙인 이유가 있을까요!? 다른 xxxData 클래스들은 id에 별다른 Validation을 안붙였더라구요 🤔

```suggestion
    var attachFiles: Map<String, ByteArrayResource> = emptyMap(),
    var id: Long = 0L
```